invenio_theme/webpack.py

```diff
@@ -38,5 +38,6 @@
         'moment': '~2.23.0',
         'select2': '~4.0.2',
         'admin-lte': '~2.4.8',
+        'morris.js': 'github:morrisjs/morris.js',
```

Comment:
Hm, we shouldn't depend on github source code checkout since it's very volatile.

Which package is using morris?

Reply:
Admin-LTE is using morris.js for some of the charts.

It doesn't appear like morris will get any new releases anytime soon. Do you have any suggestions on how to handle this?